site/public/js/admin-gradeable-updates.js

```diff
@@ -369,7 +459,7 @@ function ajaxUpdateGradeableProperty(gradeable_id, p_values, successCallback, er
                 }
                 else {
                     alert('Internal server error');
-                    console.error(response.message);
```

Comment:
@shailpatels may be able to comment more on this. Should this be reverted or is this being changed elsewhere (for other ajax requests)